conference.js

```diff
@@ -2447,36 +2449,34 @@ export default {
 
         APP.UI.removeAllListeners();
 
-        let requestFeedbackPromise;
+        let feedbackResult = true;
```

Comment:
Why init it as a boolean? Why not with type { feedbackSubmitted?: boolean; showThankYou?: boolean; wasDialogShown?: boolean;}

Reply:
i just left the previous implementation assignments. Had another look on how it was being used and updated the default